File: zad2/PamoLab/app/src/main/java/com/example/pamolab/ui/BMR/BmrViewModel.java
package com.example.pamolab.ui.BMR;

import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class BmrViewModel extends ViewModel {

    private MutableLiveData<String> mText;

    public BmrViewModel() {
        mText = new MutableLiveData<>();
        mText.setValue("Calculate BMR");
    }

    public LiveData<String> getText() {
        return mText;
    }
}

File: zad4/PamoLab/app/src/main/java/com/example/pamolab/ui/BMR/BmrViewModel.kt
package com.example.pamolab.ui.BMR

import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel

class BmrViewModel : ViewModel() {
    private val mText: MutableLiveData<String>

    init {
        mText = MutableLiveData()
        mText.value = "Calculate BMR"
    }

    val text: LiveData<String>
        get() = mText
}